ivy/functional/ivy/layers.py

```diff
@@ -916,6 +916,66 @@ def conv1d_transpose(
     ret
         The result of the transpose convolution operation.
 
+    Both the description and the type hints above assumes an array input for simplicity,
+    but this function is *nestable*, and therefore also accepts :class:`ivy.Container`
+    instances in place of any of the arguments.
```

Comment:
This is only to be mentioned in the container script's docstrings.

Reply:
I believe this is added in ivy's functional API though? The other functions and the docs don't mention it in a container's docstrings.

Reply:
Yes, you're right, my bad.